bin/kubectl-crossplane-stack-init

```diff
@@ -310,7 +319,13 @@ EOF
   fi
 }
 
-cd ${INIT_DIR}
+cd -- "${INIT_DIR}"
+
+if [ -f "${INIT_DIR}/stack.env" -a -z "${FORCE_INIT}" ]; then
```

Comment:
Would you mind moving this into a function? I have been trying to keep the main section of the script clean, using only functions. A potential name could be `check_for_existing_stack` or something similar